<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SheetWindow</class>
 <widget class="QMainWindow" name="SheetWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>515</width>
    <height>550</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SheetMeal</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout"/>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>515</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="sheetMenu">
    <property name="title">
     <string>&amp;Sheet</string>
    </property>
    <widget class="QMenu" name="recentMenu">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="title">
      <string>&amp;Recent</string>
     </property>
    </widget>
    <addaction name="newSheetAction"/>
    <addaction name="openAction"/>
    <addaction name="recentMenu"/>
    <addaction name="separator"/>
    <addaction name="saveAsAction"/>
    <addaction name="separator"/>
    <addaction name="clearSheetAction"/>
    <addaction name="refreshSheetAction"/>
    <addaction name="closeSheetAction"/>
    <addaction name="separator"/>
    <addaction name="quitAction"/>
   </widget>
   <widget class="QMenu" name="mealMenu">
    <property name="title">
     <string>&amp;Meal</string>
    </property>
    <addaction name="newMealAction"/>
    <addaction name="duplicateMealAction"/>
    <addaction name="separator"/>
    <addaction name="renameMealAction"/>
    <addaction name="clearMealAction"/>
    <addaction name="deleteMealAction"/>
    <addaction name="separator"/>
    <addaction name="addFoodAction"/>
    <addaction name="removeFoodAction"/>
   </widget>
   <widget class="QMenu" name="optionsMenu">
    <property name="title">
     <string>S&amp;ettings</string>
    </property>
    <widget class="QMenu" name="toolbarsMenu">
     <property name="title">
      <string>&amp;Toolbars</string>
     </property>
     <addaction name="sheetToolbarAction"/>
     <addaction name="mealToolbarAction"/>
    </widget>
    <addaction name="toolbarsMenu"/>
    <addaction name="separator"/>
    <addaction name="rememberSheetAction"/>
   </widget>
   <widget class="QMenu" name="helpMenu">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="instructionsAction"/>
    <addaction name="separator"/>
    <addaction name="aboutAction"/>
   </widget>
   <addaction name="sheetMenu"/>
   <addaction name="mealMenu"/>
   <addaction name="optionsMenu"/>
   <addaction name="helpMenu"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="newSheetAction">
   <property name="text">
    <string>&amp;New</string>
   </property>
   <property name="toolTip">
    <string>Create a blank sheet in a new window</string>
   </property>
  </action>
  <action name="instructionsAction">
   <property name="text">
    <string>&amp;Instructions</string>
   </property>
   <property name="toolTip">
    <string>How to use the application</string>
   </property>
  </action>
  <action name="aboutAction">
   <property name="text">
    <string>&amp;About</string>
   </property>
   <property name="toolTip">
    <string>About SheetMeal</string>
   </property>
  </action>
  <action name="openAction">
   <property name="text">
    <string>&amp;Open...</string>
   </property>
   <property name="toolTip">
    <string>Open a sheet in current window</string>
   </property>
  </action>
  <action name="saveAsAction">
   <property name="text">
    <string>&amp;Save As...</string>
   </property>
   <property name="toolTip">
    <string>Save sheet as new file</string>
   </property>
  </action>
  <action name="clearSheetAction">
   <property name="text">
    <string>&amp;Clear...</string>
   </property>
   <property name="toolTip">
    <string>Remove all meals from sheet</string>
   </property>
  </action>
  <action name="refreshSheetAction">
   <property name="text">
    <string>R&amp;efresh</string>
   </property>
   <property name="toolTip">
    <string>Reload sheet from disk</string>
   </property>
  </action>
  <action name="closeSheetAction">
   <property name="text">
    <string>C&amp;lose</string>
   </property>
   <property name="toolTip">
    <string>Close this sheet</string>
   </property>
  </action>
  <action name="quitAction">
   <property name="text">
    <string>&amp;Quit</string>
   </property>
   <property name="toolTip">
    <string>Close all sheets and exit</string>
   </property>
  </action>
  <action name="newMealAction">
   <property name="text">
    <string>&amp;New</string>
   </property>
   <property name="toolTip">
    <string>Add new meal to sheet</string>
   </property>
  </action>
  <action name="duplicateMealAction">
   <property name="text">
    <string>D&amp;uplicate</string>
   </property>
   <property name="toolTip">
    <string>Make a duplicate of meal</string>
   </property>
  </action>
  <action name="addFoodAction">
   <property name="text">
    <string>Add &amp;Food...</string>
   </property>
   <property name="toolTip">
    <string>Add food items to meal</string>
   </property>
  </action>
  <action name="clearMealAction">
   <property name="text">
    <string>&amp;Clear...</string>
   </property>
   <property name="toolTip">
    <string>Remove all food items from meal</string>
   </property>
  </action>
  <action name="deleteMealAction">
   <property name="text">
    <string>&amp;Delete...</string>
   </property>
   <property name="toolTip">
    <string>Delete meal</string>
   </property>
  </action>
  <action name="rememberSheetAction">
   <property name="text">
    <string>&amp;Remember Sheet</string>
   </property>
  </action>
  <action name="sheetToolbarAction">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Sheet</string>
   </property>
  </action>
  <action name="mealToolbarAction">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Meal</string>
   </property>
  </action>
  <action name="removeFoodAction">
   <property name="text">
    <string>&amp;Remove Food</string>
   </property>
   <property name="toolTip">
    <string>Remove selected food items</string>
   </property>
  </action>
  <action name="renameMealAction">
   <property name="text">
    <string>R&amp;ename</string>
   </property>
   <property name="toolTip">
    <string>Rename meal</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
